tests/perf-system/generator/logging_generator.py

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+from generator import Messages
+
+
+common_headers = {"host": "127.0.0.1:8000"}
+msgs = Messages()
+
+msgs.append("/app/log/private/count", "GET")
+
+for i in range(14):
+    msgs.append(
+        "/app/log/private",
+        "POST",
+        additional_headers=common_headers,
+        body='{"id": ' + str(i) + ', "msg": "Logged ' + str(i) + ' to private table"}',

```

Comment:
```suggestion
        body=f'{"id": {i}, "msg": "Logged {i} to private table"}',

```

Reply:
This is a move of `loggin_generator.py`, but I guess includes enough other changes that its confused git diff. I'll do these cleanups while we're here.